package.json

```diff
@@ -19,7 +19,7 @@
     "lint-fix": "eslint src apps/*/src tests build --ext vue --ext js --color --fix",
     "accessibility-tests": "node node_modules/axe-cli/axe-cli --exit --disable page-has-heading-one,meta-viewport",
     "accessibility-tests-drone": "node node_modules/axe-cli/axe-cli --exit --disable page-has-heading-one,meta-viewport",
-    "acceptance-tests": "cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup-local.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty tests/acceptance/features -t 'not @skip'",
+    "acceptance-tests": "cucumber-js --require-module @babel/register --require-module @babel/polyfill --require tests/acceptance/setup-local.js --require tests/acceptance/stepDefinitions --format node_modules/cucumber-pretty -t 'not @skip'",
```

Comment:
Is there a way to put logic in here so that if no folder of tests is specified then it will default to `tests/acceptance/features` ?
That might be nice, or actually might be annoying, because people will easily accidentally run the whole suite locally.